bot/bot.go

```diff
@@ -572,6 +577,15 @@ func (b *Bot) dealWithHTTPTimeouts(event *slack.MessageEvent) {
 	}
 }
 
+func (b *Bot) sliceUsageAndInternals(event *slack.MessageEvent) {
+	params := slack.PostMessageParameters{AsUser: true}
+	_, _, err := b.slackBotAPI.PostMessage(event.Channel, `<https://blog.golang.org/go-slices-usage-and-internals/>`, params)
```

Comment:
This link is "404 page not found". Did you mean https://blog.golang.org/go-slices-usage-and-internals without the trailing slash?